docs/types/mapping-types.rst

```diff
@@ -52,6 +58,34 @@ each ``_KeyType``, recursively. For example with a mapping:
         }
     }
 
+The example below uses a mapping type inside another mapping type. The important
+point to remember are that the mapping inside the mapping is not a return variable,
+but the variable inside it is.
+
+::
```

Comment:
@chriseth Myself and @ekpyron had a chat about an example for a mapping with a mapping and struggled to think of many great examples. What do you think of this one? Then I'll polish the text.

Reply:
looks good!